From 27747dcbfbde8a767b1a873716fce1d3aaf17c2f Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Fri, 16 Mar 2018 07:16:19 +0200
Subject: [PATCH] xfs_scrub: add missing paths header
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix the following build failure with musl libc:

xfs_scrub.c: In function ‘main’:
xfs_scrub.c:670:11: error: ‘_PATH_MOUNTED’ undeclared (first use in this function)
    mtab = _PATH_MOUNTED;
           ^~~~~~~~~~~~~

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream status: https://marc.info/?l=linux-xfs&m=152117771313566&w=2

 scrub/xfs_scrub.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scrub/xfs_scrub.c b/scrub/xfs_scrub.c
index ab26e6335948..eedbe34ec7b3 100644
--- a/scrub/xfs_scrub.c
+++ b/scrub/xfs_scrub.c
@@ -21,6 +21,7 @@
 #include <pthread.h>
 #include <stdbool.h>
 #include <stdlib.h>
+#include <paths.h>
 #include <sys/time.h>
 #include <sys/resource.h>
 #include <sys/statvfs.h>
-- 
2.16.2

